src/lib/resource/sections/rep-address.js

```diff
@@ -0,0 +1,185 @@
+'use strict';
+
+module.exports = {
+    section: {
+        schema: {
+            $schema: 'http://json-schema.org/draft-07/schema#',
+            type: 'object',
+            allOf: [
+                {
+                    title: 'Enter your address',
+                    meta: {
+                        compositeId: 'rep-address',
+                        classifications: {
+                            theme: 'rep-details'
+                        },
+                        summary: {
+                            title: 'Your address'
+                        }
+                    },
+                    required: ['q-rep-building-and-street', 'q-rep-town-or-city'],
+                    propertyNames: {
+                        enum: [
+                            'q-rep-building-and-street',
+                            'q-rep-building-and-street-2',
+                            'q-rep-town-or-city',
+                            'q-rep-county',
+                            'q-rep-postcode'
+                        ]
+                    },
+                    allOf: [
+                        {
+                            properties: {
+                                'q-rep-building-and-street': {
+                                    type: 'string',
+                                    title: 'Building and street',
+                                    maxLength: 60,
+                                    errorMessage: {
+                                        maxLength:
+                                            'First line of address must be less than 60 characters'
+                                    },
+                                    meta: {
+                                        classifications: {
+                                            theme: 'rep-details'
+                                        }
+                                    }
+                                }
+                            }
+                        },
+                        {
+                            properties: {
+                                'q-rep-building-and-street-2': {
+                                    type: 'string',
+                                    title: 'Building and street line 2',
+                                    maxLength: 60,
+                                    errorMessage: {
+                                        maxLength:
+                                            'Second line of address must be less than 60 characters'
```

Comment:
@armoj if max length is 60 should this be '....must be 60 characters or less'

Reply:
fixed